src/Native/src/runtime/interpreter.cpp

```diff
@@ -237,6 +238,9 @@ result<void> interpreter::run() noexcept {
                         ? ret_fields[0]
                         : tuple(std::in_place, std::move(ret_fields))));
     }
+#ifdef ENABLE_OP_PROFILE
+    op_profile::print();
```

Comment:
such modification only effect for interpreter.run, but entry.invoke is more commonly used in the 2.0 API, can't show profiled if call invoke